src/IPNFT.sol

```diff
@@ -102,9 +103,44 @@ contract IPNFT is ERC721URIStorageUpgradeable, ERC721BurnableUpgradeable, IReser
         emit Reserved(_msgSender(), reservationId);
     }
 
+    /**
+     * @notice mints an IPNFT with `tokenURI` as source of metadata. This IPNFT is linked a proof of idea (POI) which is a hash of any collection of files that represents an idea, anchored on any chain.
+     * @notice We are charging a nominal fee to symbolically represent the transfer of ownership rights, for a price of .001 ETH (<$2USD at current prices). This helps ensure the protocol is affordable to almost all projects, but discourages frivolous IP-NFT minting.
+     *
+     * @param to the recipient of the NFT
+     * @param poi the hash of the poi that will be computed to the tokenId
+     * @param _tokenURI a location that resolves to a valid IP-NFT metadata structure
+     * @param _symbol a symbol that represents the IPNFT's derivatives. Can be changed by the owner
+     * @param authorization a bytes encoded parameter that ensures that the poi is owned by the owner (to param)
+     * @return computedTokenId
+     */
+    function mintWithPOI(address to, bytes calldata poi, string calldata _tokenURI, string calldata _symbol, bytes calldata authorization)
```

Comment:
the *check* whether someone may mint an IPNFT with a poi is done off chain by the permissioner / sign off. No need to provide the "poi hash" (assumably the root metadata) right now to the minting interface. 

on *Indexers* it's pretty easily identifiable btw: you can index transactions *to* the known 1DEA address (which would be insanely usable to replace my current Alchemy approach - we can at some point even consider to add more data to the anchor transaction, eg a publicly downloadable IPFS file)